src/components/NavBar/SearchBar.tsx

```diff
@@ -105,78 +121,116 @@ export const SearchBar = () => {
     hasInput: debouncedSearchValue && debouncedSearchValue.length > 0,
     ...trace,
   }
+  const placeholderText =
+    phase1Flag === NftVariant.Enabled
+      ? isMobileOrTablet
+        ? t`Search`
+        : t`Search tokens and NFT collections`
+      : t`Search tokens`
+
+  const handleKeyPress = useCallback(
+    (event: any) => {
+      if (event.key === '/') {
+        event.preventDefault()
+        !isOpen && toggleOpen()
+      }
+    },
+    [isOpen]
+  )
+
+  useEffect(() => {
+    const innerRef = inputRef.current
+
+    if (innerRef !== null) {
+      //only mount the listener when input available as ref
+      document.addEventListener('keydown', handleKeyPress)
+    }
+
+    return () => {
+      if (innerRef !== null) {
+        document.removeEventListener('keydown', handleKeyPress)
+      }
+    }
+  }, [handleKeyPress, inputRef])
 
   return (
-    <Box position="relative">
-      <Trace section={SectionName.NAVBAR_SEARCH}>
-        <Box
-          position={{ sm: 'fixed', md: 'absolute' }}
-          width={{ sm: isOpen ? 'viewWidth' : 'auto', md: 'auto' }}
-          ref={searchRef}
-          className={isPhase1 ? styles.searchBarContainerNft : styles.searchBarContainer}
-          display={{ sm: isOpen ? 'inline-block' : 'none', xl: 'inline-block' }}
+    <Trace section={SectionName.NAVBAR_SEARCH}>
```

Comment:
good catch

Reply:
Looks like there aren't many changes below, they're just highlighted because you reordered trace as the parent element? Can you confirm?

Reply:
Yes - is that ok? Can wrap in another div if needed, but was extraneous.

Reply:
it's fine, you can use the `hide whitespace` option to see the real changes